doc/build-windows.md

```diff
@@ -83,10 +106,15 @@ Then build using:
 
 To build executables for Windows 32-bit, install the following dependencies:
 
-    sudo apt-get install g++-mingw-w64-i686 mingw-w64-i686-dev 
+    sudo apt install g++-mingw-w64-i686 mingw-w64-i686-dev 
```

Comment:
The trailing whitespace failure is the end of this line I think. Sorry for so many little reviews :) Remember to squash all your commits at the end